src/Components/AccountPageSubPages/PersonalLeaderBoard.jsx

```diff
@@ -6,12 +6,44 @@ import TableContainer from "@mui/material/TableContainer";
 import TableRow from "@mui/material/TableRow";
 import Paper from "@mui/material/Paper";
 import Grid from "@mui/material/Grid";
+import { useEffect, useState } from "react";
+
+export default function PersonalLeaderBoard({ username }) {
+  const [scoreList, setScoreList] = useState([]);
+  const [byScoreClicked, setByScoreClicked] = useState(0);
+
+  useEffect(() => {
+    const fetchData = async () => {
```

Comment:
This could go in the Networking file

Reply:
Ah I see the issue. If you are passing setScoreList into the fetchData function within useEffect, will that not still be registered as a dependency?